platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -219,19 +219,19 @@ sep1:
   ## @type: bool
   ## Whether to enable SEP-1 for this instance of AP server.
   #
-  enabled: false
+  enabled: true
   toml:
     ## @param: type
     ## @type: string
     ## @supported_values:
     ##     `string`: value contains the content of the stellar toml file
     ##     `file`: value contains the path to the stellar toml file
     ##     `url`: value contains the url to the stellar toml file
-    type: string
+    type: url
     ## @param: value
     ## @type: string
     #
-    value: |
+    value: http://169.254.169.254/latest/meta-data/
```

Comment:
We should not use the hard-coded URL in the `anchor-config-default-values.yaml`. 

If this is for test, it should be configured in the test cases instead of the default values.